packages/core/src/PasswordInput/PasswordInput.tsx

```diff
@@ -49,6 +55,9 @@ const PasswordInput: React.FC<InferProps<typeof propTypes>> = ({
 
     if (hasProgressBar) {
       regexChecks.reduce(
+        // TODO TypeScript doesn't like the return type in the reducer, so there may
+        // be a bug to fix here.
+        // @ts-ignore
```

Comment:
n-b: easy fix here 

make the function body, and return acc

Reply:
We weren't using it like a `reduce` anyway, changed to `forEach`